src/_kw

```diff
@@ -456,6 +456,7 @@ _kw_s() { _kw_ssh }
 _kw_ssh()
 {
   _arguments : \
+    '(--verbose)--verbose[enable verbose mode]' \
```

Comment:
Verbose can be used in all options.

Reply:
In this case the verbose is encompassing all options